src/pages/game/game-page.jsx

```diff
@@ -20,17 +20,20 @@ export default function GamePage({ match }) {
     setGameId(match.params.gameId);
   }, [match]);
 
-  function connectWebSocket(pid, gid) {
+  function vote(points) {
+    socket.vote(points);
+  }
+
+  function addPlayer(pid, gid, cb) {
     socket.register(gid, pid, data => {
       console.log('Got a message from the server', data);
+      setPlayerId(pid);
```

Comment:
Move these to before the socket init, we don't need to update them every time we get a message (they will never change)